src/gettingstarted/dev-environments/create-environment.md

```diff
@@ -0,0 +1,59 @@
+# Development Environments
+
+## Create environment
+
+It would be useful to test a new feature deploys correctly by building a live site of your application that includes that feature. Platform.sh supports the deployment of isolated development environments from your branches, complete with exact copies of all of your data.
+
+This is a simple example of how to set up a development environment, but if any of your features depend on the data inside your production databases, you will have full access to a complete copy of that data so that you can be sure the feature will work as expected before merging.
+
+<asciinema-player src="/videos/asciinema/branch.cast" preload=1 autoplay=1 loop=1></asciinema-player>
+
+1. **Create branch, make changes, push to Platform.sh**
+
+    First, create and checkout a branch for your new feature.
+
+    ```bash
+    git branch dev && git checkout dev
```

Comment:
I don't understand why this doesn't use the `platform environment:branch` command, thus allowing us to skip all the active/inactive discussion?